### ISSUE TITLE
Slack::Web::Client needs to be updated to support the Slack's new `blocks` parameter

### ISSUE THREAD

**Issue by gabrielmdeal** [slack-ruby/slack-ruby-client#251]
Slack just added a new `blocks` parameter to APIs like `chat_postMessage`.  The `Slack::Web::Client` APIs need to be updated to support this new parameter.  

The [slack-api-ref repo](https://github.com/slack-ruby/slack-api-ref) hasn't picked up the API change yet.  But hopefully tomorrow I or someone can do the needful: https://github.com/slack-ruby/slack-ruby-client/blob/master/CONTRIBUTING.md#update-slack-web-api

**Comment by cwise** [comment]
I'll take a crack at it.

**Comment by cwise** [comment]
Oh crud... if I had read a little more closely I would've seen the pull requests. Third times a charm.

**Comment by gabrielmdeal** [comment]
FWIW, here is a workaround:
```ruby
    blocks = JSON.dump(blocks)
    client = Slack::Web::Client.new(token: access_token)
    client.post('chat.postMessage', channel: channel_id, blocks: blocks)
```

**Comment by dblock** [comment]
I've merged #253 and will try to do a release this week.
